src/core/server/saved_objects/routes/integration_tests/migrate.test.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { migratorInstanceMock } from './migrate.test.mocks';
+import * as kbnTestServer from '../../../../../test_utils/kbn_server';
+
+describe('SavedObjects /_migrate endpoint', () => {
+  let root: ReturnType<typeof kbnTestServer.createRoot>;
+
+  beforeEach(async () => {
+    root = kbnTestServer.createRoot({ migrations: { skip: true } });
+    await root.setup();
+    await root.start();
+    migratorInstanceMock.runMigrations.mockClear();
+  }, 30000);
```

Comment:
__question:__ do we really need this explicit timeout? AFAIK we set [large enough global timeout](https://github.com/elastic/kibana/blob/master/src/dev/jest/setup/after_env.integration.js#L25) for all _integration_ jest tests unless something has changed.

Reply:
**Probably** it was added not to wait for 10 minutes to fail the test.

Reply:
:man_shrugging: if we think Kibana startup and CI became faster now we can decrease global timeout and set it to something more reasonable once and for all (out of scope of this PR, just thinking aloud).

Reply:
> Probably it was added not to wait for 10 minutes to fail the test.

Yea, that was it.

> if we think Kibana startup and CI are supposed to be faster now we can decrease global timeout and set it to something more reasonable once and for all 

+1, but changing the global timeout is not always a pleasant surprise in term of results, this is way outside of the scope of this PR.